.travis.yml

```diff
@@ -69,8 +69,9 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
-    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder,
+    # including victoria_nav_msgs and victoria_sensor_msgs
+    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs,victoria_sensor_msgs
```

Comment:
When I said that, I just meant that platformio is the actually compiler, not travis.
I thought I noticed as well that it compiled when I thought it shouldn't.
You the headers have to get into the arduino somehow.
What happens locally when you don't 
```
rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH 
```
Omitting the:
```
victoria_nav_msgs victoria_sensor_msgs
```
Does it still work? Maybe `make_libraries` can find them because ... magic?